pkg/server/k8s/client_test.go

```diff
@@ -374,7 +374,31 @@ func TestHasAnotherIngress(t *testing.T) {
 		t.Fatal("got unexpected error:", err)
 	}
 
-	if rs != false {
+	if rs != true {
 		t.Errorf("got true; want false")
```

Comment:
Could you explain this change?

Reply:
The original test was wrong 😅 . We create another ingress, so HasAnotherIngress should've returned true.

Reply:
It returned false because the k8s state wasn't persisted between function calls.

Reply:
☝️

Reply:
@drgarcia1986 I'll change the error message and the variable names, so it'll become less confusing.